Nightly build(s) fail

```
$ nodenv install nightly 
Downloading node-v8.0.0-nightly20170210b471392f8c-darwin-x64.tar.gz...
-> https://nodejs.org/download/nightly/v8.0.0-nightly20170210b471392f8c/node-v8.0.0-nightly20170210b471392f8c-darwin-x64.tar.gz
Installing node-v8.0.0-nightly20170210b471392f8c-darwin-x64...
find: /usr/local/var/nodenv/versions/nightly: No such file or directory

BUILD FAILED (OS X 10.12.3 using node-build 2.5.4)

Binary installation failed; try compiling from source with `--compile` flag

Inspect or clean up the working tree at /var/folders/sv/3941ksds4v18r9y1kyptw1jw0000gn/T/node-build.20170210213852.81278
Results logged to /var/folders/sv/3941ksds4v18r9y1kyptw1jw0000gn/T/node-build.20170210213852.81278.log

Last 10 log lines:
/var/folders/sv/3941ksds4v18r9y1kyptw1jw0000gn/T/node-build.20170210213852.81278 ~/Projects/node-build
/var/folders/sv/3941ksds4v18r9y1kyptw1jw0000gn/T/node-build.20170210213852.81278/node-v8.0.0-nightly20170210b471392f8c-darwin-x64 /var/folders/sv/3941ksds4v18r9y1kyptw1jw0000gn/T/node-build.20170210213852.81278 ~/Projects/node-build
```

The directory permissions fixing (https://github.com/nodenv/node-build/blob/3032bdf87fbddf983595e74c3db690f5a00714c0/bin/node-build#L296) is throwing an error at https://github.com/nodenv/node-build/blob/3032bdf87fbddf983595e74c3db690f5a00714c0/bin/node-build#L730 because the after_install hook has run which has moved the new node.

--- a/share/node-build/nightly
+++ b/share/node-build/nightly
@@ -1,9 +1,3 @@
-after_install_package() {
-  local v="$("${PREFIX_PATH}/bin/node" -v)"
-  local prefix="$(dirname "$PREFIX_PATH")"
-  mv "$PREFIX_PATH" "${prefix}/${v#v}"
-}
-
 nightlies="https://nodejs.org/download/nightly"
 
 read -ra manifest < <(http get "${nightlies}/index.tab" | grep src | sort -rn -t $'\t' -k2,2 -k1,1 | head -1)

